plugins/nanospell/plugin.js

```diff
@@ -887,12 +885,6 @@
 			range.extractContents().appendTo(span);
 			range.insertNode(span);
 		},
-		markTypos: function (editor, node) {
-			var range = editor.createRange();
-			range.selectNodeContents(node);
-
-			this.markTyposInRange(editor, range);
-		},
```

Comment:
I'm surprised that this could go away.  Then I noticed that in scanWords, we duplicate this functionality.